blockchain/blocks/Cargo.toml

```diff
@@ -19,7 +19,7 @@ forest_fil_types.workspace  = true
 forest_json.workspace       = true
 forest_json_utils.workspace = true
 forest_message              = { workspace = true, features = ["blst"] }
-forest_vm.workspace         = true
+
```

Comment:
```suggestion
```

Reply:
Fixed

Reply:
As of writing, this has not been fixed. Please make sure you always push all your patches before commenting.